internal/provider/provider.go

```diff
@@ -25,6 +25,68 @@ func TestAzureProvider() *schema.Provider {
 	return azureProvider(true)
 }
 
+func ValidatePartnerID(i interface{}, k string) ([]string, []error) {
+	// ValidatePartnerID checks if partner_id is any of the following:
+	//  * empty string
```

Comment:
can we change this to only validate it is a valid partner id?
```suggestion
```
and not include empty here?

Reply:
Yes, that makes complete sense. I'll commit the suggestion. Thanks @katbyte!